src/views/workflow-history/workflow-history-ungrouped-event/__tests__/workflow-history-ungrouped-event.test.tsx

```diff
@@ -80,12 +80,25 @@ jest.mock(
   })
 );
 
+jest.mock(
+  '../../workflow-history-timeline-reset-button/workflow-history-timeline-reset-button',
+  () => ({
+    __esModule: true,
+    default: ({ onReset }: { onReset: () => void }) => (
+      <button onClick={onReset} data-testid="reset-button">
+        Reset
+      </button>
+    ),
+  })
+);
+
 const mockEventInfo: WorkflowHistoryUngroupedEventInfo = {
   id: '1',
   label: 'Workflow Execution Started',
   status: 'COMPLETED',
   statusLabel: 'Completed',
   event: startWorkflowExecutionEvent,
+  canReset: false,
```

Comment:
nit: could it be an optional property?
In Go, the default value of a bool is false, and we commit to providing the value.
Is it a thing in JS/TS? Or could you support both no value and true to avoid default filed values?

Reply:
I think that's pretty easy to do, I just need to do `canReset?: boolean` in the type definition. I wanted to be more explicit, but in hindsight, negation by omission works just fine